### ISSUE TITLE
Could this be used jointly with other encodings? e.g., T5 relative position encodings

### ISSUE THREAD

**Issue by Jordy-VL** [zxilucky/LAGaBi#2]

**Comment by zxilucky** [comment]
Yes. The gaussian bias can be seamlessly combined with LayoutLMv2~v3, which have adopted the 2d T5 relative position biases, further improving their performance. Please refer to [our paper](https://aclanthology.org/2023.findings-emnlp.521/) for more details.
